core/integration/platform_test.go

```diff
@@ -184,6 +203,9 @@ func (PlatformSuite) TestCacheMounts(ctx context.Context, t *testctx.T) {
 		WithExec([]string{"sh", "-x", "-c", strings.Join(cmds, " && ")}).
 		Sync(ctx)
 	require.NoError(t, err)
+
+	cancel(cancelErr)
```

Comment:
do we also need to call `cancel` on `t.Cleanup`?

Reply:
Yep good call, fixed